ee/se/modules/380-metallb/hooks/ensure_crds.go

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
ensure_crds hook is now embedded to deckhouse controller. Delete it.

Reply:
Done